frontend/src/app/application-forms/temporary-outfitters/temp-outfitter-left-nav.component.html

```diff
@@ -1,5 +1,6 @@
-&nbsp;
-  <ul id="sidebar-nav" role="navigation" aria-label="Temp outfitter" class="usa-sidenav-list" [style.bottom]="bottom" [style.top]="top" [style.position]="position">
+<aside role="navigation" aria-label="left" class="sidebar-container">
```

Comment:
One thing I realize is that we made a shared sidebar component that didn't get back populated to the tempOutfitter, but it looks like it was for the `noncommercial-group-use/learn-more` and `http://localhost:4200/applications/temp-outfitters/learn-more`. I wonder if we want to take this opportunity to make sure that the shared sidebar component is used consistently throughout the site?

Reply:
Yeah I noticed that, I think that's a good call to try and get them all using the same component.

Reply:
It turns out there are a lot of differences between the two implementations of the sidebars right now. The sidebar component expects the content of the sidebar and main area of the page to be provided in html within an array https://github.com/18F/fs-permit-platform/blob/dev/frontend/src/app/application-forms/temporary-outfitters/temporary-outfitters-learn-more.component.ts#L11-L144 which would be difficult to make work with the form based temp outfitter component. The temp outfitter sidebar also has a considerable amount of extra logic built into it to allow for the checkmark validation as various sections of the form are completed. I think while the basic html structure of the sidebar is similar, it might be best to leave these as separate components for now.

Reply:
Sounds good, thanks for taking a look.